suite-native/intl/src/en.ts

```diff
@@ -84,7 +85,7 @@ export const en = {
         },
     },
     accountList: {
-        numberOfTokens: '+{numberOfTokens} tokens',
+        numberOfTokens: '+{numberOfTokens} Tokens',
```

Comment:
How is plurarization handled? Does it say `+1 Token` or `+1 Tokens`?

Reply:
I expected this to be handled by the task that should resolve current default +1. Added here just not to forget to capitalise first letter to make it consistent.

Do you think it should be handled here?

Reply:
OK. I don't mind, as long as it's handled somewhere. :-)